tests/api/fxa_webhooks.rs

```diff
@@ -212,21 +211,6 @@ async fn delete_user_test() -> Result<(), Error> {
     assert_eq!(json["geo"]["country"], "Iceland");
     assert_eq!(json["is_authenticated"], true);
 
-    let base_url = "/api/v1/plus/collection/?url=/en-US/docs/Web/CSS";
```

Comment:
Can we instead of deleting this update it to v2. I think we added this to test that we cascade delete things. (maybe add a comment)